astapi_app  | INFO:     172.21.0.1:37488 - "GET /favicon.ico HTTP/1.1" 404 Not Found
postgres_db  | 2025-03-11 17:51:39.545 UTC [35] FATAL:  password authentication failed for user "postgres"
postgres_db  | 2025-03-11 17:51:39.545 UTC [35] DETAIL:  Password does not match for user "postgres".
fastapi_app  | INFO:     172.21.0.1:37488 - "POST /auth/token HTTP/1.1" 500 Internal Server Error                                                                                                                                                   
postgres_db  |  Connection matched pg_hba.conf line 99: "host all all all md5"                                                                                                                                                                      
fastapi_app  | ERROR:    Exception in ASGI application
fastapi_app  | Traceback (most recent call last):                                                                                                                                                                                                   
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/uvicorn/protocols/http/h11_impl.py", line 403, in run_asgi                                                                                                                           
fastapi_app  |     result = await app(  # type: ignore[func-returns-value]                                                                                                                                                                          
fastapi_app  |              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^                                                                                                                                                                          
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/uvicorn/middleware/proxy_headers.py", line 60, in __call__                                                                                                                           
fastapi_app  |     return await self.app(scope, receive, send)                                                                                                                                                                                      
fastapi_app  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^                                                                                                                                                                                      
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/fastapi/applications.py", line 1054, in __call__                                                                                                                                     
fastapi_app  |     await super().__call__(scope, receive, send)                                                                                                                                                                                     
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/starlette/applications.py", line 112, in __call__                                                                                                                                    
fastapi_app  |     await self.middleware_stack(scope, receive, send)                                                                                                                                                                                
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/starlette/middleware/errors.py", line 187, in __call__
fastapi_app  |     raise exc                                                                                                                                                                                                                        
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/starlette/middleware/errors.py", line 165, in __call__                                                                                                                               
fastapi_app  |     await self.app(scope, receive, _send)                                                                                                                                                                                            
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/starlette/middleware/exceptions.py", line 62, in __call__                                                                                                                            
fastapi_app  |     await wrap_app_handling_exceptions(self.app, conn)(scope, receive, send)                                                                                                                                                         
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/starlette/_exception_handler.py", line 53, in wrapped_app                                                                                                                            
fastapi_app  |     raise exc
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/starlette/_exception_handler.py", line 42, in wrapped_app                                                                                                                            
fastapi_app  |     await app(scope, receive, sender)                                                                                                                                                                                                
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/starlette/routing.py", line 714, in __call__                                                                                                                                         
fastapi_app  |     await self.middleware_stack(scope, receive, send)                                                                                                                                                                                
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/starlette/routing.py", line 734, in app                                                                                                                                              
fastapi_app  |     await route.handle(scope, receive, send)
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/starlette/routing.py", line 288, in handle                                                                                                                                           
fastapi_app  |     await self.app(scope, receive, send)                                                                                                                                                                                             
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/starlette/routing.py", line 76, in app                                                                                                                                               
fastapi_app  |     await wrap_app_handling_exceptions(app, request)(scope, receive, send)                                                                                                                                                           
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/starlette/_exception_handler.py", line 53, in wrapped_app                                                                                                                            
fastapi_app  |     raise exc                                                                                                                                                                                                                        
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/starlette/_exception_handler.py", line 42, in wrapped_app
fastapi_app  |     await app(scope, receive, sender)                                                                                                                                                                                                
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/starlette/routing.py", line 73, in app                                                                                                                                               
fastapi_app  |     response = await f(request)                                                                                                                                                                                                      
fastapi_app  |                ^^^^^^^^^^^^^^^^                                                                                                                                                                                                      
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/fastapi/routing.py", line 301, in app                                                                                                                                                
fastapi_app  |     raw_response = await run_endpoint_function(                                                                                                                                                                                      
fastapi_app  |                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^                                                                                                                                                                                      
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/fastapi/routing.py", line 212, in run_endpoint_function
fastapi_app  |     return await dependant.call(**values)                                                                                                                                                                                            
fastapi_app  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^                                                                                                                                                                                            
fastapi_app  |   File "/app/src/auth/routers.py", line 44, in login_for_access_token                                                                                                                                                                
fastapi_app  |     user = await user_repo.get_user_by_username(form_data.username)                                                                                                                                                                  
fastapi_app  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^                                                                                                                                                                  
fastapi_app  |   File "/app/src/auth/repos.py", line 34, in get_user_by_username                                                                                                                                                                    
fastapi_app  |     result = await self.session.execute(query)
fastapi_app  |              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^                                                                                                                                                                                       
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/sqlalchemy/ext/asyncio/session.py", line 463, in execute                                                                                                                             
fastapi_app  |     result = await greenlet_spawn(                                                                                                                                                                                                   
fastapi_app  |              ^^^^^^^^^^^^^^^^^^^^^                                                                                                                                                                                                   
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/sqlalchemy/util/_concurrency_py3k.py", line 201, in greenlet_spawn                                                                                                                   
fastapi_app  |     result = context.throw(*sys.exc_info())                                                                                                                                                                                          
fastapi_app  |              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/sqlalchemy/orm/session.py", line 2365, in execute                                                                                                                                    
fastapi_app  |     return self._execute_internal(                                                                                                                                                                                                   
fastapi_app  |            ^^^^^^^^^^^^^^^^^^^^^^^                                                                                                                                                                                                   
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/sqlalchemy/orm/session.py", line 2241, in _execute_internal                                                                                                                          
fastapi_app  |     conn = self._connection_for_bind(bind)                                                                                                                                                                                           
fastapi_app  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^                                                                                                                                                                                           
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/sqlalchemy/orm/session.py", line 2110, in _connection_for_bind                                                                                                                       
fastapi_app  |     return trans._connection_for_bind(engine, execution_options)                                                                                                                                                                     
fastapi_app  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
fastapi_app  |   File "<string>", line 2, in _connection_for_bind                                                                                                                                                                                   
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/sqlalchemy/orm/state_changes.py", line 139, in _go                                                                                                                                   
fastapi_app  |     ret_value = fn(self, *arg, **kw)                                                                                                                                                                                                 
fastapi_app  |                 ^^^^^^^^^^^^^^^^^^^^                                                                                                                                                                                                 
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/sqlalchemy/orm/session.py", line 1189, in _connection_for_bind                                                                                                                       
fastapi_app  |     conn = bind.connect()
fastapi_app  |            ^^^^^^^^^^^^^^                                                                                                                                                                                                            
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/sqlalchemy/engine/base.py", line 3274, in connect                                                                                                                                    
fastapi_app  |     return self._connection_cls(self)                                                                                                                                                                                                
fastapi_app  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^                                                                                                                                                                                                
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/sqlalchemy/engine/base.py", line 146, in __init__                                                                                                                                    
fastapi_app  |     self._dbapi_connection = engine.raw_connection()                                                                                                                                                                                 
fastapi_app  |                              ^^^^^^^^^^^^^^^^^^^^^^^
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/sqlalchemy/engine/base.py", line 3298, in raw_connection                                                                                                                             
fastapi_app  |     return self.pool.connect()                                                                                                                                                                                                       
fastapi_app  |            ^^^^^^^^^^^^^^^^^^^                                                                                                                                                                                                       
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/sqlalchemy/pool/base.py", line 449, in connect                                                                                                                                       
fastapi_app  |     return _ConnectionFairy._checkout(self)                                                                                                                                                                                          
fastapi_app  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^                                                                                                                                                                                          
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/sqlalchemy/pool/base.py", line 1263, in _checkout
fastapi_app  |     fairy = _ConnectionRecord.checkout(pool)                                                                                                                                                                                         
fastapi_app  |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^                                                                                                                                                                                         
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/sqlalchemy/pool/base.py", line 712, in checkout                                                                                                                                      
fastapi_app  |     rec = pool._do_get()                                                                                                                                                                                                             
fastapi_app  |           ^^^^^^^^^^^^^^                                                                                                                                                                                                             
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/sqlalchemy/pool/impl.py", line 179, in _do_get                                                                                                                                       
fastapi_app  |     with util.safe_reraise():                                                                                                                                                                                                        
fastapi_app  |          ^^^^^^^^^^^^^^^^^^^
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/sqlalchemy/util/langhelpers.py", line 146, in __exit__                                                                                                                               
fastapi_app  |     raise exc_value.with_traceback(exc_tb)                                                                                                                                                                                           
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/sqlalchemy/pool/impl.py", line 177, in _do_get                                                                                                                                       
fastapi_app  |     return self._create_connection()                                                                                                                                                                                                 
fastapi_app  |            ^^^^^^^^^^^^^^^^^^^^^^^^^                                                                                                                                                                                                 
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/sqlalchemy/pool/base.py", line 390, in _create_connection                                                                                                                            
fastapi_app  |     return _ConnectionRecord(self)
fastapi_app  |            ^^^^^^^^^^^^^^^^^^^^^^^                                                                                                                                                                                                   
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/sqlalchemy/pool/base.py", line 674, in __init__                                                                                                                                      
fastapi_app  |     self.__connect()                                                                                                                                                                                                                 
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/sqlalchemy/pool/base.py", line 900, in __connect                                                                                                                                     
fastapi_app  |     with util.safe_reraise():                                                                                                                                                                                                        
fastapi_app  |          ^^^^^^^^^^^^^^^^^^^                                                                                                                                                                                                         
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/sqlalchemy/util/langhelpers.py", line 146, in __exit__
fastapi_app  |     raise exc_value.with_traceback(exc_tb)                                                                                                                                                                                           
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/sqlalchemy/pool/base.py", line 896, in __connect                                                                                                                                     
fastapi_app  |     self.dbapi_connection = connection = pool._invoke_creator(self)                                                                                                                                                                  
fastapi_app  |                                          ^^^^^^^^^^^^^^^^^^^^^^^^^^                                                                                                                                                                  
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/sqlalchemy/engine/create.py", line 646, in connect                                                                                                                                   
fastapi_app  |     return dialect.connect(*cargs, **cparams)
fastapi_app  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^                                                                                                                                                                                        
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/sqlalchemy/engine/default.py", line 622, in connect                                                                                                                                  
fastapi_app  |     return self.loaded_dbapi.connect(*cargs, **cparams)                                                                                                                                                                              
fastapi_app  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^                                                                                                                                                                              
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/sqlalchemy/dialects/postgresql/asyncpg.py", line 961, in connect                                                                                                                     
fastapi_app  |     await_only(creator_fn(*arg, **kw)),                                                                                                                                                                                              
fastapi_app  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/sqlalchemy/util/_concurrency_py3k.py", line 132, in await_only                                                                                                                       
fastapi_app  |     return current.parent.switch(awaitable)  # type: ignore[no-any-return,attr-defined] # noqa: E501                                                                                                                                 
fastapi_app  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^                                                                                                                                                                                          
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/sqlalchemy/util/_concurrency_py3k.py", line 196, in greenlet_spawn                                                                                                                   
fastapi_app  |     value = await result                                                                                                                                                                                                             
fastapi_app  |             ^^^^^^^^^^^^                                                                                                                                                                                                             
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/asyncpg/connection.py", line 2421, in connect
fastapi_app  |     return await connect_utils._connect(                                                                                                                                                                                             
fastapi_app  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^                                                                                                                                                                                             
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/asyncpg/connect_utils.py", line 1049, in _connect                                                                                                                                    
fastapi_app  |     conn = await _connect_addr(                                                                                                                                                                                                      
fastapi_app  |            ^^^^^^^^^^^^^^^^^^^^                                                                                                                                                                                                      
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/asyncpg/connect_utils.py", line 886, in _connect_addr                                                                                                                                
fastapi_app  |     return await __connect_addr(params, True, *args)
fastapi_app  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^                                                                                                                                                                                 
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/asyncpg/connect_utils.py", line 934, in __connect_addr                                                                                                                               
fastapi_app  |     await connected                                                                                                                                                                                                                  
fastapi_app  | asyncpg.exceptions.InvalidPasswordError: password authentication failed for user "postgres"                                                                                                                                          
postgres_db  | 2025-03-11 17:51:52.097 UTC [36] FATAL:  password authentication failed for user "postgres"                                                                                                                                          
postgres_db  | 2025-03-11 17:51:52.097 UTC [36] DETAIL:  Password does not match for user "postgres".
fastapi_app  | INFO:     172.21.0.1:46506 - "POST /auth/register HTTP/1.1" 500 Internal Server Error                                                                                                                                                
postgres_db  |  Connection matched pg_hba.conf line 99: "host all all all md5"
fastapi_app  | ERROR:    Exception in ASGI application                                                                                                                                                                                              
fastapi_app  | Traceback (most recent call last):                                                                                                                                                                                                   
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/uvicorn/protocols/http/h11_impl.py", line 403, in run_asgi                                                                                                                           
fastapi_app  |     result = await app(  # type: ignore[func-returns-value]                                                                                                                                                                          
fastapi_app  |              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^                                                                                                                                                                          
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/uvicorn/middleware/proxy_headers.py", line 60, in __call__                                                                                                                           
fastapi_app  |     return await self.app(scope, receive, send)
fastapi_app  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^                                                                                                                                                                                      
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/fastapi/applications.py", line 1054, in __call__                                                                                                                                     
fastapi_app  |     await super().__call__(scope, receive, send)                                                                                                                                                                                     
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/starlette/applications.py", line 112, in __call__                                                                                                                                    
fastapi_app  |     await self.middleware_stack(scope, receive, send)                                                                                                                                                                                
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/starlette/middleware/errors.py", line 187, in __call__
fastapi_app  |     raise exc                                                                                                                                                                                                                        
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/starlette/middleware/errors.py", line 165, in __call__                                                                                                                               
fastapi_app  |     await self.app(scope, receive, _send)                                                                                                                                                                                            
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/starlette/middleware/exceptions.py", line 62, in __call__                                                                                                                            
fastapi_app  |     await wrap_app_handling_exceptions(self.app, conn)(scope, receive, send)                                                                                                                                                         
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/starlette/_exception_handler.py", line 53, in wrapped_app
fastapi_app  |     raise exc                                                                                                                                                                                                                        
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/starlette/_exception_handler.py", line 42, in wrapped_app                                                                                                                            
fastapi_app  |     await app(scope, receive, sender)                                                                                                                                                                                                
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/starlette/routing.py", line 714, in __call__                                                                                                                                         
fastapi_app  |     await self.middleware_stack(scope, receive, send)                                                                                                                                                                                
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/starlette/routing.py", line 734, in app
fastapi_app  |     await route.handle(scope, receive, send)                                                                                                                                                                                         
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/starlette/routing.py", line 288, in handle                                                                                                                                           
fastapi_app  |     await self.app(scope, receive, send)                                                                                                                                                                                             
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/starlette/routing.py", line 76, in app                                                                                                                                               
fastapi_app  |     await wrap_app_handling_exceptions(app, request)(scope, receive, send)                                                                                                                                                           
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/starlette/_exception_handler.py", line 53, in wrapped_app                                                                                                                            
fastapi_app  |     raise exc
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/starlette/_exception_handler.py", line 42, in wrapped_app                                                                                                                            
fastapi_app  |     await app(scope, receive, sender)                                                                                                                                                                                                
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/starlette/routing.py", line 73, in app                                                                                                                                               
fastapi_app  |     response = await f(request)                                                                                                                                                                                                      
fastapi_app  |                ^^^^^^^^^^^^^^^^                                                                                                                                                                                                      
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/fastapi/routing.py", line 301, in app                                                                                                                                                
fastapi_app  |     raw_response = await run_endpoint_function(
fastapi_app  |                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^                                                                                                                                                                                      
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/fastapi/routing.py", line 212, in run_endpoint_function                                                                                                                              
fastapi_app  |     return await dependant.call(**values)                                                                                                                                                                                            
fastapi_app  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^                                                                                                                                                                                            
fastapi_app  |   File "/app/src/auth/routers.py", line 32, in register                                                                                                                                                                              
fastapi_app  |     existing_user = await user_repo.get_user_by_email(user_create.email)                                                                                                                                                             
fastapi_app  |                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^                                                                                                                                                             
fastapi_app  |   File "/app/src/auth/repos.py", line 29, in get_user_by_email
fastapi_app  |     result = await self.session.execute(query)                                                                                                                                                                                       
fastapi_app  |              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^                                                                                                                                                                                       
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/sqlalchemy/ext/asyncio/session.py", line 463, in execute                                                                                                                             
fastapi_app  |     result = await greenlet_spawn(                                                                                                                                                                                                   
fastapi_app  |              ^^^^^^^^^^^^^^^^^^^^^                                                                                                                                                                                                   
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/sqlalchemy/util/_concurrency_py3k.py", line 201, in greenlet_spawn                                                                                                                   
fastapi_app  |     result = context.throw(*sys.exc_info())                                                                                                                                                                                          
fastapi_app  |              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/sqlalchemy/orm/session.py", line 2365, in execute                                                                                                                                    
fastapi_app  |     return self._execute_internal(                                                                                                                                                                                                   
fastapi_app  |            ^^^^^^^^^^^^^^^^^^^^^^^                                                                                                                                                                                                   
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/sqlalchemy/orm/session.py", line 2241, in _execute_internal                                                                                                                          
fastapi_app  |     conn = self._connection_for_bind(bind)                                                                                                                                                                                           
fastapi_app  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^                                                                                                                                                                                           
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/sqlalchemy/orm/session.py", line 2110, in _connection_for_bind                                                                                                                       
fastapi_app  |     return trans._connection_for_bind(engine, execution_options)                                                                                                                                                                     
fastapi_app  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
fastapi_app  |   File "<string>", line 2, in _connection_for_bind                                                                                                                                                                                   
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/sqlalchemy/orm/state_changes.py", line 139, in _go                                                                                                                                   
fastapi_app  |     ret_value = fn(self, *arg, **kw)                                                                                                                                                                                                 
fastapi_app  |                 ^^^^^^^^^^^^^^^^^^^^                                                                                                                                                                                                 
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/sqlalchemy/orm/session.py", line 1189, in _connection_for_bind                                                                                                                       
fastapi_app  |     conn = bind.connect()                                                                                                                                                                                                            
fastapi_app  |            ^^^^^^^^^^^^^^                                                                                                                                                                                                            
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/sqlalchemy/engine/base.py", line 3274, in connect                                                                                                                                    
fastapi_app  |     return self._connection_cls(self)                                                                                                                                                                                                
fastapi_app  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^                                                                                                                                                                                                
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/sqlalchemy/engine/base.py", line 146, in __init__                                                                                                                                    
fastapi_app  |     self._dbapi_connection = engine.raw_connection()                                                                                                                                                                                 
fastapi_app  |                              ^^^^^^^^^^^^^^^^^^^^^^^                                                                                                                                                                                 
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/sqlalchemy/engine/base.py", line 3298, in raw_connection
fastapi_app  |     return self.pool.connect()                                                                                                                                                                                                       
fastapi_app  |            ^^^^^^^^^^^^^^^^^^^                                                                                                                                                                                                       
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/sqlalchemy/pool/base.py", line 449, in connect                                                                                                                                       
fastapi_app  |     return _ConnectionFairy._checkout(self)                                                                                                                                                                                          
fastapi_app  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^                                                                                                                                                                                          
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/sqlalchemy/pool/base.py", line 1263, in _checkout                                                                                                                                    
fastapi_app  |     fairy = _ConnectionRecord.checkout(pool)
fastapi_app  |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^                                                                                                                                                                                         
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/sqlalchemy/pool/base.py", line 712, in checkout                                                                                                                                      
fastapi_app  |     rec = pool._do_get()                                                                                                                                                                                                             
fastapi_app  |           ^^^^^^^^^^^^^^                                                                                                                                                                                                             
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/sqlalchemy/pool/impl.py", line 179, in _do_get                                                                                                                                       
fastapi_app  |     with util.safe_reraise():                                                                                                                                                                                                        
fastapi_app  |          ^^^^^^^^^^^^^^^^^^^
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/sqlalchemy/util/langhelpers.py", line 146, in __exit__                                                                                                                               
fastapi_app  |     raise exc_value.with_traceback(exc_tb)                                                                                                                                                                                           
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/sqlalchemy/pool/impl.py", line 177, in _do_get                                                                                                                                       
fastapi_app  |     return self._create_connection()                                                                                                                                                                                                 
fastapi_app  |            ^^^^^^^^^^^^^^^^^^^^^^^^^                                                                                                                                                                                                 
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/sqlalchemy/pool/base.py", line 390, in _create_connection                                                                                                                            
fastapi_app  |     return _ConnectionRecord(self)
fastapi_app  |            ^^^^^^^^^^^^^^^^^^^^^^^                                                                                                                                                                                                   
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/sqlalchemy/pool/base.py", line 674, in __init__                                                                                                                                      
fastapi_app  |     self.__connect()                                                                                                                                                                                                                 
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/sqlalchemy/pool/base.py", line 900, in __connect                                                                                                                                     
fastapi_app  |     with util.safe_reraise():                                                                                                                                                                                                        
fastapi_app  |          ^^^^^^^^^^^^^^^^^^^                                                                                                                                                                                                         
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/sqlalchemy/util/langhelpers.py", line 146, in __exit__
fastapi_app  |     raise exc_value.with_traceback(exc_tb)                                                                                                                                                                                           
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/sqlalchemy/pool/base.py", line 896, in __connect                                                                                                                                     
fastapi_app  |     self.dbapi_connection = connection = pool._invoke_creator(self)                                                                                                                                                                  
fastapi_app  |                                          ^^^^^^^^^^^^^^^^^^^^^^^^^^                                                                                                                                                                  
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/sqlalchemy/engine/create.py", line 646, in connect                                                                                                                                   
fastapi_app  |     return dialect.connect(*cargs, **cparams)                                                                                                                                                                                        
fastapi_app  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^                                                                                                                                                                                        
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/sqlalchemy/engine/default.py", line 622, in connect                                                                                                                                  
fastapi_app  |     return self.loaded_dbapi.connect(*cargs, **cparams)                                                                                                                                                                              
fastapi_app  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^                                                                                                                                                                              
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/sqlalchemy/dialects/postgresql/asyncpg.py", line 961, in connect                                                                                                                     
fastapi_app  |     await_only(creator_fn(*arg, **kw)),                                                                                                                                                                                              
fastapi_app  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/sqlalchemy/util/_concurrency_py3k.py", line 132, in await_only                                                                                                                       
fastapi_app  |     return current.parent.switch(awaitable)  # type: ignore[no-any-return,attr-defined] # noqa: E501                                                                                                                                 
fastapi_app  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^                                                                                                                                                                                          
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/sqlalchemy/util/_concurrency_py3k.py", line 196, in greenlet_spawn                                                                                                                   
fastapi_app  |     value = await result                                                                                                                                                                                                             
fastapi_app  |             ^^^^^^^^^^^^                                                                                                                                                                                                             
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/asyncpg/connection.py", line 2421, in connect                                                                                                                                        
fastapi_app  |     return await connect_utils._connect(
fastapi_app  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^                                                                                                                                                                                             
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/asyncpg/connect_utils.py", line 1049, in _connect                                                                                                                                    
fastapi_app  |     conn = await _connect_addr(                                                                                                                                                                                                      
fastapi_app  |            ^^^^^^^^^^^^^^^^^^^^                                                                                                                                                                                                      
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/asyncpg/connect_utils.py", line 886, in _connect_addr                                                                                                                                
fastapi_app  |     return await __connect_addr(params, True, *args)                                                                                                                                                                                 
fastapi_app  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^                                                                                                                                                                                 
fastapi_app  |   File "/usr/local/lib/python3.12/site-packages/asyncpg/connect_utils.py", line 934, in __connect_addr                                                                                                                               
fastapi_app  |     await connected
fastapi_app  | asyncpg.exceptions.InvalidPasswordError: password authentication failed for user "postgres"  